src/helpers/tests/geoChart/common.spec.ts

```diff
@@ -124,4 +125,78 @@ describe("common", () => {
             },
         );
     });
+
+    describe("getGeoPushpinBucketsInMdObject", () => {
+        const attributeGeo1 = {
+            visualizationAttribute: {
+                localIdentifier: "localIdentifier123",
+                displayForm: {
+                    uri: "/gdc/displayform/geo1",
+                },
+            },
+        };
+        const attributeGeo2 = {
+            visualizationAttribute: {
+                localIdentifier: "localIdentifier456",
+                displayForm: {
+                    uri: "/gdc/displayform/geo2",
+                },
+            },
+        };
+
+        it("should return buckets has toottip text when geo pushpin has location item", () => {
+            const mdObject = {
+                visualizationClass: {
+                    uri: "/gdc/visualizationclass/1",
+                },
+                buckets: [
+                    {
+                        localIdentifier: "location",
+                        items: [attributeGeo1],
+                    },
+                ],
+                tooltipText: {
+                    localIdentifier: "tooltipText",
+                    items: [attributeGeo2],
+                },
+            };
+            const buckets = getGeoPushpinBucketsInMdObject(mdObject);
```

Comment:
`buckets` is not a suitable name.
I suggest to change it to `updatedMdObject`, `newMdObject` or something like that :)

Reply:
updated